scripts/apps/authoring/authoring/directives/ArticleEditDirective.js

```diff
@@ -319,6 +321,40 @@ export function ArticleEditDirective(
                 });
             };
 
+            /**
```

Comment:
If you're gonna use the `/**` form of comments, then please use the standard format, like other places in our codebase. You can get inspired from other files:
https://github.com/superdesk/superdesk-client-core/blob/2e5cf281023293d7d4e1799394549759c8aabcf7/scripts/core/services/storage.js#L1-L25 or [here](https://github.com/superdesk/superdesk-client-core/blob/368c2295e3f5cf1b4bb66c911d94b8234b5e0f7e/scripts/apps/archive/services/MultiService.js) 

Or, if you want to use the shorter form, just use:
```js
// Shows the preview for the given embed field.
```
Without the `/**` notation.